peering/models/models.py

```diff
@@ -531,9 +532,21 @@ class DirectPeeringSession(BGPSession):
     router = models.ForeignKey(
         "Router", blank=True, null=True, on_delete=models.SET_NULL
     )
+    service_reference = models.CharField(
+        max_length=255,
+        unique=True,
+        blank=True,
+        null=True,
+        help_text="Optional: Internal Service Reference (will auto generate if left blank)",
+    )
 
     class Meta(BGPSession.Meta):
-        ordering = ["local_autonomous_system", "autonomous_system", "ip_address"]
+        ordering = [
+            "service_reference",
```

Comment:
Maybe use the reference field as value to define the string representation of a BGP session.

Reply:
I suspect by this you mean returning `self.service_reference` from `__str__()` within BGPSession? If this is the case, are you aware of anywhere else we may use this in its current string representation (e.g. populating HTML data with the current string representation of the model?)

Reply:
Exactly. This is used for instance as the title of object views for peering sessions. More generally, it will be used as soon as we want to display a session as a string.